Src/HALALMock/Services/ADC/ADC.cpp

```diff
@@ -77,38 +105,68 @@ void ADC::turn_on(uint8_t id){
 		ErrorHandler("LPTIM - %d - of ADC - %d - did not start correctly", timer.name, peripheral->handle);
 		return;
 	}
+*/ // Commented out because of the mock
```

Comment:
Remove commented code if it is not used, the code is already on the original, no need to keep it here